src/analyzer.ts

```diff
@@ -211,7 +227,10 @@ export class Analyzer {
             return this._scanInlineDocument(
                 scannedDependency, document, warnings);
           } else if (scannedDependency instanceof ScannedImport) {
-            return this._scanImport(scannedDependency, warnings);
+            if (scannedDependency.type !== 'lazy-html-import') {
+              return this._scanImport(scannedDependency, warnings);
```

Comment:
add a TODO: I think this will need to be configurable. Some cases will not want lazy features. Pretty sure this isn't the right place for it (it should probably be on the querying side), but seems like an ok place for the TODO